config/content_items.yml

```diff
@@ -16,11 +16,3 @@ special_routes:
     base_path: "/account/home"
     title: "Account home page"
     rendering_app: "frontend"
-  - content_id: "e5098fc2-ad79-4c6f-882b-410831c12f60"
-    base_path: "/account/saved-pages/add"
-    title: "Save a page"
-    rendering_app: "frontend"
-  - content_id: "3aeaeb46-b3c5-449b-8a3b-2753ffef3c9f"
-    base_path: "/account/saved-pages/remove"
-    title: "Remove a saved page"
-    rendering_app: "frontend"
```

Comment:
We'll want to unpublish these content items

Reply:
Yeah was going to ask about those - This is just a draft for now until the tests pass

Reply:
Cool this is now ready for a proper look, will confirm here when i've unpublished in all 3 envs

Reply:
I've unpublished these as I needed to publish the special route for `/sign-in/redirect`